htdocs/main.css

```diff
@@ -31,7 +31,7 @@ html, body {
 }
 
 #page {
-    min-height: 100%;
+    min-height: -webkit-fill-available;
```

Comment:
have you tried adding a min-height on the html/body or setting the min-height in vh instead of adding a webkit-only hack?

Reply:
yeah i only just realized that webkit isn't supported on lets say, firefox

Reply:
what's vh?

Reply:
okay got it! the issue isn't html/body, both are at 100% height. it's the "page" div that gets cut because of the footer underneath it

Reply:
percentage is relative to the parent. The previous CSS is 100% of the parent height, and I think the problem is that the parent isn't 100% of the size of the window (because its height is 100%, not its min-height)

Reply:
it goes 
```
<html>
  <body>
    <div id="wrap">
      <div id="page">
```

body has height 100%, wrap has min-height 100% (no height because otherwise footer would hover over everything), and page has min-height 100%. i agree with you that page's height should become 100% but it doesn't for some reason

Reply:
@driusan setting min-height to 100 vh seems to be the best option. is this still not a good idea?

Reply:
100vh is fine, as far as I know most browsers support css3 units.